src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__links {
+    display: flex;
+    font-size: 12px;
+    line-height: 14px;
+    margin: 0;
+    padding: 0;
```

Comment:
let's give this job to resetting styles, not here

Reply:
agree